<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionWindow</class>
 <widget class="QMainWindow" name="QuestionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="answer1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>361</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 20px;
	font-weight:700;
	color: rgb(43, 45, 135);
	background:rgb(244, 178, 0) ;
}</string>
    </property>
    <property name="text">
     <string>Nothing</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background: rgb(46, 49, 146);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="questionText">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>691</width>
       <height>201</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Text Semibold</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: whitesmoke;
}</string>
     </property>
     <property name="text">
      <string>Which ocean borders the west coast of the United States?</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="questionIndex">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>271</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Britannic Bold</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: whitesmoke;
}</string>
     </property>
     <property name="text">
      <string>Question 1</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="answer3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>320</y>
       <width>401</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 20px;
	font-weight:700;
	color: rgb(43, 45, 135);
	background:rgb(244, 178, 0) ;
}</string>
     </property>
     <property name="text">
      <string>Nothing</string>
     </property>
    </widget>
    <widget class="QPushButton" name="answer2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>361</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 20px;
	font-weight:700;
	color: rgb(43, 45, 135);
	background:rgb(244, 178, 0) ;
}</string>
     </property>
     <property name="text">
      <string>Nothing</string>
     </property>
    </widget>
    <widget class="QPushButton" name="answer4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>430</y>
       <width>401</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 20px;
	font-weight:700;
	color: rgb(43, 45, 135);
	background:rgb(244, 178, 0) ;
}</string>
     </property>
     <property name="text">
      <string>Nothing</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_username">
        <property name="font">
         <font>
          <family>Sitka Subheading</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: whitesmoke;
}</string>
        </property>
        <property name="text">
         <string>User   :</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_score">
        <property name="font">
         <font>
          <family>Sitka Subheading</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: whitesmoke;
}</string>
        </property>
        <property name="text">
         <string>Score   :</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_difficulty">
        <property name="font">
         <font>
          <family>Sitka Subheading</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: whitesmoke;
}</string>
        </property>
        <property name="text">
         <string>Difficulty   :</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>answer1</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
